eWonConnector-gateway/src/main/java/org/imdc/ewon/SyncManager.java

```diff
@@ -698,6 +737,30 @@ protected void updateTagValues(EwonData data) {
             BasicScanclassHistorySet historySet =
                     new BasicScanclassHistorySet(providerName, "_exempt_", -1);
 
+            // Add AllRealtime tag under Ewon _config tag folder
+            if (!registeredEwons.contains(device))
+            {
+                // Create path to Ewon's all realtime tag
+                String ewonRealtimePath = buildTagPath(device + "/_config/AllRealtime");
+
+                // Add Ewon to registered Ewon list
+                registeredEwons.add(device);
+
+                // Register all realtime tag write handler with provider
+                provider.registerWriteHandler(ewonRealtimePath, new WriteHandler() {
```

Comment:
Was it necessary to create a write handler for this tag? I would think you would be able to do it without registering a write handler.

Probably a more efficient approach is to use the write handler to register/unregister ewons in the realtimeEwonList.  That way you could eliminate an async read.

Reply:
Yeah it was. I tried without, and Ignition gave an error that it was not writable.

Also, that's a good idea. I can update the code and push it to this PR.

Reply:
@alexjhawk, I do have concerns that if the ignition sdk misses that event (for whatever reason).  That ewon will not be read realtime.  I cant think of a case where this would happen but given how many other strange things happen its not unreasonable.  Worse case is that the user has to deselect it then select it again if its not working.

Reply:
Concerns with the write handler or the efficient way?

Reply:
Concerns with the write handler.  If for some reason the event doesnt get triggered when the data is changed you will basically be stuck in the state you dont want to be in until the value is changed again.

This theoretically should not happen but if it does occasionally happen its a little difficult to resolve without checking the value the async read way.

Reply:
Oh ok. How often does that happen?

Reply:
It theoretically never happens.

Reply:
write handler now adds/removes from a set, and fetchRealtimeTags just checks for Ewon to be in the set.